packages/namada.nix

```diff
@@ -0,0 +1,34 @@
+{
+  pkgs,
+  namada-src,
+}:
+pkgs.rustPlatform.buildRustPackage {
+  pname = "namada";
+  version = "v0.27.0";
+  src = namada-src;
+  nativeBuildInputs = with pkgs;
+    if stdenv.isLinux
+    then [pkg-config protobuf llvmPackages.libclang llvmPackages.libcxxClang clang]
+    else [darwin.apple_sdk.frameworks.Security];
```

Comment:
I'm surprised linux is so different from mac, I guess the mac env has the correct llvm / c packages? Does it not need protobufs?

Reply:
Oh I still wasn't able to have it work on Linux so I still haven't tested on Mac. I suspect I might need to change the `protobuf` to be used for both.

Reply:
Maybe Nix uses the default LLVM toolchain present on Mac via Xcode/Command-Line Tools?